components/arm/fake/fake.go

```diff
@@ -208,24 +207,19 @@ func (a *Arm) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
-// CurrentInputs TODO.
+// CurrentInputs returns the current inputs of the fake arm.
 func (a *Arm) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
-	res, err := a.JointPositions(ctx, nil)
-	if err != nil {
-		return nil, err
-	}
-	return a.model.InputFromProtobuf(res), nil
+	a.mu.RLock()
+	defer a.mu.RUnlock()
+	return a.joints, nil
 }
 
-// GoToInputs TODO.
+// GoToInputs moves the fake arm to the given inputs.
 func (a *Arm) GoToInputs(ctx context.Context, inputSteps ...[]referenceframe.Input) error {
 	for _, goal := range inputSteps {
 		a.mu.RLock()
 		positionDegs := a.model.ProtobufFromInput(goal)
 		a.mu.RUnlock()
-		if err := arm.CheckDesiredJointPositions(ctx, a, goal); err != nil {
```

Comment:
I think we still want this, no?

Reply:
Actually never mind; for a `fake` arm it shouldn't be possible to get slightly OOB like a real arm.